_episodes/02-make-it-public.md

```diff
@@ -58,21 +58,46 @@ on this topic, and other advice on starting an open source project, check out th
     - This file is automatically displayed on the front page of your repository
     - It has important information about your project
 
- To format the contents of this file you will use Markdown syntax. If you want to know more about it you can use this [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) or this [short guide](https://guides.github.com/features/mastering-markdown/)
-
-
+ The '.md' extension indicates that the file is written in Markdown language, which uses plain text formatting syntax that can be converted into web output format. Markdown syntax is basically plain text with a few non-alphabetic characters such as * or # for bold, italics, headings, etc. to style test. For example, you can make some text bold by adding two asterisks \** before and after the text: 
```

Comment:
Small fix: 

> to style test

 should probably be 

> to style text

Reply:
Thank you @dbampalikis for your contribution. I like the more description of makdown, but I don't agree with removing the links to the guides. Can you merge those two?

Reply:
@mkuzak The pull request is from @allegravia , I only added a comment about a misspelled word.

Reply:
yes, you're right, thank you for reviewing :)